server/app/assets/javascripts/admin_predicate_configuration.ts

```diff
@@ -72,72 +72,62 @@ class AdminPredicateConfiguration {
     let hasValueMissing = false
 
     // Check if any scalars are missing a value.
-    Array.from(
-      document.querySelectorAll<HTMLSelectElement>('.cf-scalar-select select'),
-    ).forEach((el: HTMLSelectElement) => {
-      if (el.options[el.options.selectedIndex].value == '') {
-        hasSelectionMissing = true
-      }
-    })
+    document
+      .querySelectorAll<HTMLSelectElement>('.cf-scalar-select select')
+      .forEach((el: HTMLSelectElement) => {
+        if (el.options[el.options.selectedIndex].value == '') {
+          hasSelectionMissing = true
+        }
+      })
 
     // Check if any operators are missing a value.
-    Array.from(
-      document.querySelectorAll<HTMLSelectElement>(
-        '.cf-operator-select select',
-      ),
-    ).forEach((el: HTMLSelectElement) => {
-      if (el.options[el.options.selectedIndex].value == '') {
-        hasSelectionMissing = true
-      }
-    })
+    document
+      .querySelectorAll<HTMLSelectElement>('.cf-operator-select select')
+      .forEach((el: HTMLSelectElement) => {
+        if (el.options[el.options.selectedIndex].value == '') {
+          hasSelectionMissing = true
+        }
+      })
 
     // Check if any inputs are missing a value.
     document
-      .querySelector('#predicate-config-value-row-container')
```

Comment:
nit: #predicate-config-value-row-container name isn't very clear, I'm not sure if this is a convention or not but if it isn't is it possible to rename to something clearer? Feel free to ignore if my request is naive and the cost outweighs the benefits.

Reply:
Thanks for the suggestion! Based on the naming throughout the page this name actually does make decent sense to me - this page is for predicate config, and value rows are the rows of value input boxes (vs the scalar, operator, etc. inputs at the top), and this contains all of those. The term value row is used widely throughout the page and associated utils (like for testing). I'm happy to consider if you have a specific suggestion, but I don't have a better one myself.